packages/material-ui-styles/src/makeStyles/makeStyles.js

```diff
@@ -201,7 +201,7 @@ export default function makeStyles(stylesOrCreator, options = {}) {
     classNamePrefix,
   };
 
-  return (props = {}) => {
+  const useStyles = (props = {}) => {
```

Comment:
![Screenshot from 2020-02-02 00-44-29](https://user-images.githubusercontent.com/12292047/73600761-40cdfe80-4555-11ea-897c-567d993aedbc.png)
vs.
![Screenshot from 2020-02-02 00-44-10](https://user-images.githubusercontent.com/12292047/73600764-43305880-4555-11ea-8005-00d701fdbf0b.png)

We could potentially incorporate the `name` option into the display name of the hook but that might be confusing if the hook name changes between components. Usually you only have a single `useStyles` per component and even if you have more than 1 then the `name` will be reflected in the debug value.